lib/components/DateFilter/DateFilter.js

```diff
@@ -0,0 +1,65 @@
+import { useCallback } from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  Accordion,
+  FilterAccordionHeader,
+} from '@folio/stripes/components';
+import { DateRangeFilter as DateTimeRangeFilter } from '@folio/stripes/smart-components';
```

Comment:
Please keep the original import name, it'll be easier to find all instances of the component